frontend/src/metabase/admin/settings/selectors.js

```diff
@@ -511,3 +516,9 @@ export const getActiveSection = createSelector(
     }
   },
 );
+
+export const getXraysEnabled = createSelector(getSettingValues, settings => {
```

Comment:
I'd put this in `frontend/src/metabase/selectors/settings.js` with `getIsPublicSharingEnabled` etc. It also needs to be added to `public-settings` in `src/metabase/public_settings.clj`